### PR TITLE
[PPP-4447] - Upgraded jackrabbit-core to 2.16.5 to match the server a…

### PR DESCRIPTION
**Pull request by wseyler**
…nd resolve issues with Tika vulnerabilities.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -76,7 +76,7 @@
     <jcr.version>2.0</jcr.version>
     <maven-surefire-plugin.reuseForks>false</maven-surefire-plugin.reuseForks>
     <ehcache-core.version>2.5.1</ehcache-core.version>
-    <jackrabbit-core.version>2.14.2</jackrabbit-core.version>
+    <jackrabbit-core.version>2.16.5</jackrabbit-core.version>
```
Comment (pentaho-lmartins) [inline pom.xml]: Please move the version property to the parent pom repository and use dependency management.

**Comment by buildguy** [comment]
### :gear: Build finished in 4m 20s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit
```
:ok_hand: All tests passed!
 ---

|Total|Failed|Skipped|
|---|---|---|
|343|0|0|

|[Details](http://wingman2.pentaho.net/job/projects/job/data-access-plugin/job/PR-1074/2/testReport)|
|----|


:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 19s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit
```
:exclamation: No tests found!


:information_source:  _This is an automatic message_

File: pom.xml (hunk 1 above)
Comment (wseyler) [reply pom.xml]: Done.

**Comment by wseyler** [comment]
@pentaho-lmartins This PR can't build because of upstream changes to maven-parent-poms.

File: pom.xml [hunk 2]
```diff
@@ -1265,7 +1264,7 @@
       <dependency>
         <groupId>org.apache.jackrabbit</groupId>
         <artifactId>jackrabbit-core</artifactId>
-        <version>${jackrabbit-core.version}</version>
+        <version>${jackrabbit.version}</version>
```
Comment (pentaho-lmartins) [inline pom.xml]: Please use dependency management in the parent POM.
All references to versions should only exist in the parent POM.

**Comment by buildguy** [comment]
### :gear: Build finished in 16s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit
```
:exclamation: No tests found!


:information_source:  _This is an automatic message_

File: pom.xml (hunk 2 above)
Comment (wseyler) [reply pom.xml]: Because of child dependency management tags this file required a reference to jackrabbit.version.

File: pom.xml [hunk 3]
```diff
@@ -1265,7 +1264,6 @@
       <dependency>
         <groupId>org.apache.jackrabbit</groupId>
         <artifactId>jackrabbit-core</artifactId>
-        <version>${jackrabbit-core.version}</version>
         <scope>test</scope>
       </dependency>
```
Comment (pentaho-lmartins) [inline pom.xml]: This is unnecessary given that you're moving dependency management to the parent POM in https://github.com/pentaho/maven-parent-poms/pull/172

**Comment by pentaho-lmartins** [comment]
Closing in favor of #1076.

**Pull request closed without merging**